widget/textgrid.go

```diff
@@ -444,65 +703,65 @@ func (t *textGridRenderer) addCellsIfRequired() {
 	}
 }
 
-func (t *textGridRenderer) refreshGrid() {
-	line := 1
+func (t *textGridRowRenderer) refreshCells() {
 	x := 0
+	if t.obj.row >= len(t.obj.text.text.Rows) {
+		return // empty?
```

Comment:
The question mark in the comment makes me worried. Is there something strange going on here? Are unsure about that the effect of returning here does?